docs/src/app/navigation.jsx

```diff
@@ -121,6 +134,7 @@ class CoreAppNavigation extends React.PureComponent {
 
         this.onLogoClick = this.onLogoClick.bind(this);
         this.toggleNavigation = this.toggleNavigation.bind(this);
+        this.listJSX = this.renderList();
```

Comment:
This is maybe a good idea for all of these arrays of columns and such, as long as they are indeed static/constant (i.e. they do not change based on props or state, etc.)  Saves having to "recompute" them on every render.